### DISCUSSION TITLE
icons without nvim-web-devicons

### DISCUSSION THREAD

**Discussion by jessexknight** [nvim-tree/nvim-tree.lua#2476] [Q&A]
(total nvim/lua noob here, plz have mercy)

The [homepage](https://github.com/nvim-tree/nvim-tree.lua) says that `nvim-web-devicons` is optional. Great - I don't want to use it. But I don't know how to (easily) configure any alternate (e.g., ASCII) icons. Right now I get strange symbols due to the missing font. I was expecting a simple toggle like "don't use web-devicons" which defaults to ASCII icons.

Thanks,

**Comment by alex-courtis** [comment]
See `:help nvim-tree.renderer.icons.web_devicons` to disable web-devicons icons.

See `:help nvim-tree.renderer.icons.glyphs` to configure text icons to replace the defaults.
